2-copy-of-code/lesson-15/scripts/checkout/paymentSummary.js

```diff
@@ -19,13 +19,19 @@ export function renderPaymentSummary() {
   const taxCents = totalBeforeTaxCents * 0.1;
   const totalCents = totalBeforeTaxCents + taxCents;
 
+  let cartQuantity = 0;
```

Comment:
I don't understand why we had to calculate the cartQuantity again when we could have just imported the calculateCartQuantity() from from cart.js like this: 

import { cart, calculateCartQuantity } from '../../data/cart.js';

<div>Items (${calculateCartQuantity()}):</div>

I just tried like like above, and it works.

Reply:
That's correct... How did you update the paymentSummary after modifying the quantity? did you use recursion?

Reply:
If you want your paymentSummary to be updated, whenever you edit the quantity of product on you cart.
- go to orderSummary.js, we added an EventListener on the savelink
- on the last line add the renderPaymentSummary( )

This way, your paymentSummary will be updated, when you modify the quantity and click the savelink.

Reply:
> I don't understand why we had to calculate the cartQuantity again when we could have just imported the calculateCartQuantity() function from from cart.js like this:
> 
> import { cart, calculateCartQuantity } from '../../data/cart.js';
> 
> Items (${calculateCartQuantity()}):
> I just tried like like above, and it works.

Since calculateCartQuantity() was part of Lesson 14 exercise, It is not present in Lesson 15 code hence need to calculate quantity again. If you are continuing from Lesson 14 code than it will work